### PR TITLE
feat(corner-player): support now playing

### PR DESCRIPTION
**Pull request by colmugx**
### Description

There is nothing in 'now playing', so we can use the `MediaSession API` to provide a more robust media playback experience.

Here is the effect when we used `MediaSession`

image

And something likes this when used the `Sketchybar`

image

### Additional context

There are some problems or troubles here:

- I'am not sure if this is the best place for this code.
- `cover processor`(I apologize for temporarily referring to it) is in `<FeedIcon />`, I suggest that we isolate this logic to more elegantly support `Now Playing`.

### CODE REVIEW

File: apps/renderer/src/modules/feed-column/corner-player.tsx [hunk 1]
```diff
@@ -102,6 +108,20 @@ const CornerPlayerImpl = () => {
     scopes: HotKeyScopeMap.Home,
   })
 
+  useEffect(() => {
+    if (!entry || !feed) {
+      return noop
```
Comment (Innei) [inline apps/renderer/src/modules/feed-column/corner-player.tsx:113]: Why?

File: apps/renderer/src/modules/feed-column/corner-player.tsx [hunk 2]
```diff
@@ -34,6 +36,9 @@ export const CornerPlayer = () => {
   const entry = useEntry(entryId)
   const feed = useFeedById(entry?.feedId)
 
+  //FIXME: I am not sure
```
Comment (Innei) [inline apps/renderer/src/modules/feed-column/corner-player.tsx:39]: Remove this

File: apps/renderer/src/atoms/media-session.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { atom } from "jotai"
+import { useEffect } from "react"
+
+import { createAtomHooks } from "~/lib/jotai"
+
+export const [, , useMediaMetadataValue, useMediaMetadataSetValue] = createAtomHooks(
```
Comment (Innei) [inline apps/renderer/src/atoms/media-session.ts:6]: It doesn't look like a global state manager is needed, so remove this

File: apps/renderer/src/modules/feed-column/corner-player.tsx [hunk 4]
```diff
@@ -102,6 +108,20 @@ const CornerPlayerImpl = () => {
     scopes: HotKeyScopeMap.Home,
   })
 
+  useEffect(() => {
+    if (!entry || !feed) {
+      return noop
+    }
+
+    mediaMetadataSetValue(
```
Comment (Innei) [inline apps/renderer/src/modules/feed-column/corner-player.tsx:116]: Not suitable, can be put in `Player` management method

**Review by Innei — CHANGES_REQUESTED** [review]

**Comment by colmugx** [comment]
@Innei Absolutely! Yeah, it makes sense.

There's a document about [MediaSession](https://developer.mozilla.org/en-US/docs/Web/API/MediaSession). Besides metadata setup, it offers system media control handlers. It's the reason that I've structured this as an `atom` to support future updates.

But i removed it. and i wasn't sure if I understand all the logic of the player...

**Review by Innei — APPROVED** [review]
Nice implemention

**Pull request merged**